test/lib/ex_aws/kms/integration_test.exs

```diff
@@ -0,0 +1,107 @@
+defmodule ExAws.KMSIntegratinTest do
+  use ExUnit.Case, async: true
+
+  defp key_id do
+    System.get_env("AWS_KEY_ARN")
+  end
```

Comment:
Change it https://github.com/CargoSense/ex_aws/pull/280/commits/a7c60c3263b4e7dc944b5d59bd4ac8886090943d